ddd/word.rb

```diff
@@ -1,19 +1,12 @@
-require 'set'
+require_relative 'set_of_words'
```

Comment:
You don't have to require this here, because you're not using the `SetOfWords` class in this file...

Reply:
true true